events/urls.py

```diff
@@ -1,8 +1,10 @@
-from django.urls import path
-
+from django.urls import path, re_path
 from events import views
+from checkins import views as checkin_views
 
 urlpatterns = [
-    path('events', views.EventList.as_view(), name='events'),
+    re_path(r'^events/$', views.EventList.as_view(), name='events'),
```

Comment:
Better to replace with `path('events/', views.EventList.as_view(), name='events'),` since we do not currently use regex url paths anywhere else :)